module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,32 @@
+#include "sort.hpp"
+
+std::deque<std::string> lengthSort(const std::forward_list<std::string>& list) {
+
+    std::deque<std::string> deq{};
+    auto listIt = list.begin();
+
+    if (!list.empty()) {
+        deq.insert(deq.begin(), *listIt);
+        listIt++;
+    }
```

Comment:
You can initialize deque using iterators: `std::deque<std::string> deq{list.begin(), list.end()};

Reply:
done